launcher/ui/instanceview/AccessibleInstanceView.cpp

```diff
@@ -279,7 +279,7 @@ bool AccessibleInstanceView::unselectRow(int row)
     QItemSelection selection(index, index);
     auto selectionModel = view()->selectionModel();
 
-    switch (view()->selectionMode()) {
+    switch (const auto selectionType = view()->selectionMode()) {
```

Comment:
This goes along with the below suggestion
```suggestion
    switch (view()->selectionMode()) {
```

Reply:
No longer required